op-chain-ops/genesis/genesis.go

```diff
@@ -91,21 +92,11 @@ func NewL2Genesis(config *DeployConfig, l1StartHeader *types.Header) (*core.Gene
 		difficulty = newHexBig(0)
 	}
 
-	extraData := config.L2GenesisBlockExtraData
-	if extraData == nil {
-		// L2GenesisBlockExtraData is optional, so use a default value when nil
-		extraData = BedrockTransitionBlockExtraData
-	}
-	// Ensure that the extradata is valid
-	if size := len(extraData); size > 32 {
-		return nil, fmt.Errorf("transition block extradata too long: %d", size)
-	}
-
 	genesis := &core.Genesis{
 		Config:     &optimismChainConfig,
 		Nonce:      uint64(config.L2GenesisBlockNonce),
 		Timestamp:  l1StartTime,
-		ExtraData:  extraData,
+		ExtraData:  HoloceneExtraData,
```

Comment:
wondering why below in line 111 we check `IsEcotone` if Ecotone (and following forks, including Holocene) are always active.

Reply:
Its just easier to always put the holocene extradata in but we could conditionally put it in if we really wanted to, it just adds a footgun if we have configurable extradata

Reply:
looks like no good reason!  simplified.